test/environment_unittest.cc

```diff
@@ -42,6 +49,59 @@ class TemporaryFile {
  private:
   boost::filesystem::path path_;
 };
+
+// Starts a server in a separate thread, allowing it to choose an
+// available port.
+class FakeServerThread {
```

Comment:
Why not leave it as `FakeServer`? The fact that it launches a separate thread is an implementation detail.

Reply:
Done.